website/content/en/preview/getting-started/_index.md

```diff
@@ -13,3 +13,241 @@ Learn more about Karpenter and how to get started below.
 * [Karpenter EKS Best Practices](https://aws.github.io/aws-eks-best-practices/karpenter/) guide
 * [EC2 Spot Workshop for Karpenter](https://ec2spotworkshops.com/karpenter.html)
 * [EKS Karpenter Workshop](https://www.eksworkshop.com/beginner/085_scaling_karpenter/set_up_the_provisioner/)
+
+This guide shows how to get started with Karpenter by installing and configuring it on a Kubernetes cluster.
+If that cluster is currently using Cluster Autoscaler, the guide also gives you the option of migrating away from Cluster Autoscaler.
+
+This guide assumes that you are starting with an existing Kubernetes cluster and that the following are true:
+
+* You will use existing VPC and subnets
+* You will use existing security groups
+* Your workloads have pod disruption budgets that adhere to [EKS best practices](https://aws.github.io/aws-eks-best-practices/karpenter/)
+* Your cluster has an [OIDC provider](https://docs.aws.amazon.com/eks/latest/userguide/enable-iam-roles-for-service-accounts.html) for service accounts
+* Your nodes may be part of one or more node groups
+
+This guide will also assume you have the `aws` and `kubectl` CLI tools installed.
+You can also perform many of these steps in the console, but we will use the command line for simplicity.
+
+## Get a cluster
+To use Karpenter, you must be running a Kubernetes cluster on a supported cloud provider.
+Currently, only AWS is supported.
```

Comment:
Thoughts on scoping this guide to just karpenter on EKS and not mention other options or lack thereof -- this will help the content survive the test of time.

Reply:
Okay. How should we handle kOps then? Should I just make it clear that the steps provided in this guide are only supported for EKS? That way we could still mention the kOps procedure, since it handles adding Karpenter itself. Or should we just drop the link to kOps?

Reply:
Okay. I changed the text to say that only EKS on AWS is currently supported. I said that if they wanted to try Karpenter on unsupported Kubernetes distributions on AWS, they could try kOps. I left the link to kOps instructions in.